### PR TITLE
[backport stable-5.0] tests: add docker hub authentication in jobs

### PR DESCRIPTION
**Pull request by guits**
This commit makes all jobs authenticating to docker hub in order to
avoid the rate limit.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>
(cherry picked from commit 40307f810c76f22b7152cb1f4113089a22a84274)

### CODE REVIEW

File: tox-update.ini [hunk 1]
```diff
@@ -52,8 +52,11 @@ commands=
       fetch_directory={env:FETCH_DIRECTORY:{changedir}/fetch} \
       ceph_stable_release={env:CEPH_STABLE_RELEASE:octopus} \
       ceph_docker_registry={env:CEPH_DOCKER_REGISTRY:docker.io} \
-      ceph_docker_image={env:CEPH_DOCKER_IMAGE:ceph/daemon} \
-      ceph_docker_image_tag={env:CEPH_DOCKER_IMAGE_TAG:latest-octopus} \
+      ceph_docker_image={env:UPDATE_CEPH_DOCKER_IMAGE:ceph/daemon} \
+      ceph_docker_image_tag={env:UPDATE_CEPH_DOCKER_IMAGE_TAG:latest-octopus} \
```
Comment (dsavineau) [inline tox-update.ini:56]: Why changing `CEPH_DOCKER_IMAGE` to `UPDATE_CEPH_DOCKER_IMAGE` ?

I know we have the same value at the moment but we will have to revert the change when using nautilus for the initial deployment.

WDYT ?

**Comment by guits** [comment]
jenkins test centos-container-lvm_osds

**Comment by guits** [comment]
jenkins test centos-container-lvm_batch
